node/src/components/consensus/highway_core/active_validator.rs

```diff
@@ -88,7 +88,7 @@ impl<C: Context> ActiveValidator<C> {
             vidx,
             secret,
             next_round_exp: state.params().init_round_exp(),
-            next_timer: Timestamp::zero(),
```

Comment:
Hmm... why was this zero before?

Reply:
I think that was by mistake; but I guess it would have caused a timer to fire right away, and then we'd just have requested the next timer, with the right timestamp.